lib/plugins/features/movies/controller.js

```diff
@@ -7,3 +7,34 @@ exports.create = async (payload) => {
 
   return new Movie({ id: movie.id }).fetch();
 };
+
+exports.retrieve = async (queryParams) => {
+  return new Movie().query((qb) => {
+
+    if (queryParams) {
```

Comment:
Would it be possible to flatten these if statements a bit by short-circuiting this check to return immediately if `!queryParams`?

Reply:
Yes! Good suggestion!